config/samples/dataplane_v1beta1_openstackdataplanenodeset_bgp_ovn_cluster.yaml

```diff
@@ -0,0 +1,174 @@
+apiVersion: dataplane.openstack.org/v1beta1
+kind: OpenStackDataPlaneNodeSet
+metadata:
+  name: openstack-edpm
+spec:
+  env:
+    - name: ANSIBLE_FORCE_COLOR
+      value: "True"
+  services:
+    - download-cache
+    - bootstrap
+    - configure-network
+    - validate-network
+    - frr
+    - install-os
+    - configure-os
+    - run-os
+    - ovn
+    - neutron-metadata
+    - ovn-bgp-agent
+    - libvirt
+    - nova
+    - telemetry
+  preProvisioned: true
+  nodes:
+      edpm-compute-0:
+        hostName: edpm-compute-0
+        ansible:
+          ansibleHost: 192.168.122.100
+          ansibleVars:
+            ctlplane_ip: 192.168.122.100
+            internal_api_ip: 172.17.0.100
+            storage_ip: 172.18.0.100
+            tenant_ip: 172.19.0.100
+            fqdn_internal_api: edpm-compute-0.example.com
+  networkAttachments:
+    - ctlplane
+  nodeTemplate:
+    ansibleSSHPrivateKeySecret: dataplane-ansible-ssh-private-key-secret
+    managementNetwork: ctlplane
+    ansible:
+      ansibleUser: cloud-admin
+      ansiblePort: 22
+      ansibleVars:
+         service_net_map:
+           nova_api_network: internal_api
+           nova_libvirt_network: internal_api
```

Comment:
I know other samples have this, but we need to clean all of them up too. There is nothing that uses `service_net_map` though. So we can drop this.